plugins/pencilblue/templates/admin/sites/site_form.html

```diff
@@ -35,6 +35,13 @@
                         </select>
                         <div class="form_error" ng-if="defaultLocale==''">^loc_REQUIRED_FIELD^</div>
                     </div>
+                    <div class="form-group"> 
+                        <label>Force Locale</label> 
+                        <div class="btn-group"> 
+                            <button type="button" class="btn btn-default" ng-class="{'active': forceLocale}" ng-click="forceLocale = true">^loc_YES^</button>
```

Comment:
Coming in #884 is a deprecation warning for localization flags that aren't fully qualified.  Please update these to 
```
^loc_generic.YES^
^loc_generic.NO^
```